README.md

```diff
@@ -29,3 +29,18 @@ See some of the many HDFS tutorials out there for more details, but here's a qui
 1. `hadoop fs -ls hdfs://<ActiveNameNode>:50071/` should show nothing for starters
 2. `hadoop fs -put /path/to/src_file hdfs://<ActiveNameNode>:50071/`
 3. `hadoop fs -ls hdfs://<ActiveNameNode>:50071/` should now list src_file
+
+Resource Reservation Instructions (Optional)
+--------------------------
+
+1. On master, add the role for HDFS, by running `echo hdfs > /etc/mesos-master/role`
+2. Then restart the master by running `sudo service mesos-master restart`
+3. On each slave where you want to reserve resources, add specific resource reservations for the HDFS role. Here is one example:
+<br>`cpus(*):2;cpus(hdfs):2;mem(*):4096;mem(hdfs):4096 > /etc/mesos-slave/resources`</br>
+4. On each slave with the new settings, stop the mesos slave by running
+<br>`sudo service mesos-slave stop`</br>
+5. On each slave with the new settings, remove the old slave state by running
+<br>`rm -f /tmp/mesos/meta/slaves/latest`</br>
```

Comment:
IMPORTANT: This removes the slave's checkpointed state, so the restarted slave will not know about any executors/tasks that are still running. Executors from checkpointing frameworks will not exit until after the `--recovery_timeout` flag (default 15min), so you may want to manually kill your tasks as well. The upcoming maintenance primitives will provide a cleaner interface for this.